bindings/java/src/main/com/apple/foundationdb/Tenant.java

```diff
@@ -303,11 +305,13 @@ default CompletableFuture<Void> waitPurgeGranulesComplete(byte[] purgeKey) {
 	 *
 	 * @param purgeKey key to watch
 	 * @param e the {@link Executor} to use for asynchronous callbacks
+
+	 * @return -- undocumented
```

Comment:
It returns void here. If it returns successfully, the data has been purged tracked by the provided purgeKey from purgeBlobGranules().